src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/entity/actionplan/StepEntity.kt

```diff
@@ -30,49 +29,45 @@ import java.util.UUID
 @Table(name = "step")
 @Entity
 @EntityListeners(AuditingEntityListener::class)
-class StepEntity(
-  @Id
-  @GeneratedValue
-  @UuidGenerator
-  var id: UUID? = null,
-
+data class StepEntity(
```

Comment:
Goal and Step are so closely related it made sense to do both of them together in the same PR